### PR TITLE
feat: add naver webtoon

### PR DESCRIPTION
**Pull request by zfanta**
https://comic.naver.com

### CODE REVIEW

File: lib/routes/webtoons/naver.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const domain = 'https://comic.naver.com';
+
+async function getDescription(link) {
+    const { body } = await got.get(link);
+    const $ = cheerio.load(body);
+    return $('#comic_view_area > div.wt_viewer').html();
+}
+
+async function getItems($) {
+    return Promise.all(
```
Comment (DIYgod) [inline lib/routes/webtoons/naver.js]: 没有异步操作，不需要使用 Promise.all

**Comment by zfanta** [comment]
I added docs.

File: lib/routes/webtoons/naver.js (hunk 1 above)
Comment (zfanta) [reply lib/routes/webtoons/naver.js]: At line 16 callback of map has async function. So it needs Promise.all. If Promise.all is removed title and description in all items would be undefined.

Comment (DIYgod) [reply lib/routes/webtoons/naver.js]: Sorry, I miss it.

File: lib/routes/webtoons/naver.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const domain = 'https://comic.naver.com';
+
+async function getDescription(link) {
+    const { body } = await got.get(link);
```
Comment (DIYgod) [inline lib/routes/webtoons/naver.js]: should use cache for invariable content here, see: https://docs.rsshub.app/en/joinus/#enable-caching

Comment (zfanta) [reply lib/routes/webtoons/naver.js]: I fixed it. please check it.

**Pull request merged**